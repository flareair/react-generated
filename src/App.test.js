import React from 'react';
import App from './App';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import configureMockStore from 'redux-mock-store'

let dataStore = null;

beforeEach(() => {
  dataStore = configureMockStore([]);
});

describe('App component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Provider store={dataStore()}>
        <App />
      </Provider>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
