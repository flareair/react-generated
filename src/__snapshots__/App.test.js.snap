// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders correctly 1`] = `
<div
  className="container"
>
  <header>
    <h1 />
  </header>
  <div
    className="counter"
  >
    <p>
      Count: 
    </p>
    <div
      aria-label="controls"
      className="btn-group"
      role="group"
    >
      <button
        className="btn btn-primary"
        onClick={[Function]}
      >
        Increase
      </button>
      <button
        className="btn btn-warning"
        onClick={[Function]}
      >
        Decrease
      </button>
      <button
        className="btn btn-default"
        onClick={[Function]}
      >
        Reset
      </button>
    </div>
  </div>
</div>
`;
